data_structures/heap.cpp

```diff
@@ -0,0 +1,68 @@
+#include<iostream>
+
+using namespace std;
+
+void heapify(int arr[], int n, int i)
```

Comment:
Functions should be documented explaining the use of its parameters, a brief description, and what it returns.